src/region.js

```diff
@@ -315,42 +236,39 @@ var Region = MarionetteObject.extend({
     if (view.destroy) {
       view.destroy();
     } else {
+      const shouldTriggerDetach = !!view._isAttached;
```

Comment:
I like the idea of making this self-documenting.  `_destroyView()` is almost 100% `_destroyBackboneView()` anyway, so I renamed `_destroyView()` -> `_destroyBackboneView()` and pulled the decision to call it up to `empty()`.  (Next commit)